### ISSUE TITLE
extract inverse transform

### ISSUE THREAD

**Issue by Biomiha** [RGLab/CytoML#150]
Hi,

Is there a way of extracting the inverse tranform from a flowjo workspace when the `execute = FALSE` argument is used in flowjo_to_gatingset

    gs <- flowjo_to_gatingset(ws, name = 4, execute = FALSE, additional.keys = NULL)

I followed this workflow: 
https://bioc.ism.ac.jp/packages/3.12/bioc/vignettes/CytoML/inst/doc/flowjo_to_gatingset.html#2_Advanced

`gh_get_transformations(gs[[1]], channel = "B710-A")` gives me the tranformations, but is there a direct way of obtaining the inverse transform? 
Basically, I would just like to be able to use `axis_x_inverse_trans()` to give me a sensible scale when plotting.

Many thanks in advance,
Miha
